@Etelis related test failures:
```
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierMultiBlock::test_store_multiple_blocks - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierMultiBlock::test_partial_block_lookup - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierFailures::test_lookup_exception_returns_false - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierFailures::test_submit_store_register_memory_failure_reported_in_get_finished - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierFailures::test_submit_load_register_memory_failure_reported_in_get_finished - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierFailures::test_submit_store_make_prepped_xfer_failure_reported_in_get_finished - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierFailures::test_failure_and_success_both_returned_by_get_finished - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierShutdown::test_shutdown_clears_in_flight_transfers - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] FAILED v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierShutdown::test_shutdown_idempotent - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_lookup_empty_tier - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_store_and_lookup - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_lookup_unrelated_key_returns_false - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_store_then_load_roundtrip - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_multiple_jobs_tracked_independently - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_failed_transfer_reported - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_pending_transfer_not_returned_until_done - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
[2026-06-18T12:59:03Z] ERROR v1/kv_offload/tiering/test_obj_tier.py::TestMockObjTierBasic::test_drain_jobs_polls_until_transfers_complete - AttributeError: 'types.SimpleNamespace' object has no attribute 'use_v2_model_runner'
```

--- a/tests/v1/kv_offload/tiering/test_obj_tier.py
+++ b/tests/v1/kv_offload/tiering/test_obj_tier.py
@@ -37,6 +37,7 @@ def _make_vllm_config():
             decode_context_parallel_size=1,
             rank=0,
         ),
+        use_v2_model_runner=False,
     )
 
 

